int32 movement # positive -> forward | negative -> backward | 0 = STOP
---
bool success
string info
